src/components/email-signup-form/index.tsx

```diff
@@ -18,7 +18,7 @@ const EmailSignupForm = () => {
     setMessage("");
     e.preventDefault();
 
-    const endpoint = "https://api.uofthacks.com/xi/email_list/add";
+    const endpoint = "https://api.uofthacks.com/xii/email_list/add";
```

Comment:
mb can you change this to 12? 

`https://api.uofthacks.com/12/email_list/add` then should be good to merge